openedx/core/djangoapps/profile_images/views.py

```diff
@@ -35,49 +35,75 @@ def _make_upload_dt():
+        The requesting user must be signed in.  Users can only add profile
+        images to their own account.  Users with staff access can remove
+        profile images for other user accounts.  All other user can
+        only remove their own profile images.
 
-    **Example Request**
+    **Example Requests**
 
-        POST /api/profile_images/v1/{username}/upload
+        POST /api/user/v1/accounts/{username}/image
 
-    **Example Responses**
+        DELETE /api/user/v1/accounts/{username}/image
 
-        When the requesting user tries to upload the image for a different user, the
-        request returns one of the following responses.
+    **Example POST Responses**
 
-        * If the requesting user has staff access, the request returns an HTTP 403
-          "Forbidden" response.
+        * If the upload could not be performed, the request returns an HTTP 400
+          "Bad Request" response with more information.
 
-        * If the requesting user does not have staff access, the request returns
-          an HTTP 404 "Not Found" response.
+        * If the upload is successful, the request returns an HTTP 204 "No
+          Content" response with no additional content.
 
-        * If no user matches the "username" parameter, the request returns an HTTP
-          404 "Not Found" response.
+        When the requesting user tries to upload the image for a different
+        user, the request may return any of the following:
 
-        * If the upload could not be performed, the request returns an HTTP 400 "Bad
-          Request" response with more information.
+        * If no user matches the "username" parameter, the request returns an
+          HTTP 404 "Not Found" response.
 
-        * If the upload is successful, the request returns an HTTP 204 "No Content"
-          response with no additional content.
+        * If the user exists and the requesting user does not have staff
+          access, the request returns an HTTP 404 "Not Found" response.
+
+        * If the user exists and the requesting user has staff access, the
+          request returns an HTTP 403 "Forbidden" response.
 
+    **Example DELETE Responses**
+
+        * If the image could not be removed, the request returns an HTTP 400
+          "Bad Request" response with more information.
+
+        * If the request successfully removes the image, the request returns
+          an HTTP 204 "No Content" response with no additional content.
+
+        When the requesting user tries to remove the profile image for a
+        different user, the view may also return one of the following
+        responses:
+
+        * If the user does not have staff access, the request returns an HTTP
+          404 "Not Found" response.
+
+        * If no user matches the "username" parameter, the request returns an
+          HTTP 404 "Not Found" response.
+
+        * If a user matches the "username" parameter and the user has staff
```

Comment:
Nit: Although outside the scope of this PR, it would be nice to keep the permission details outside of the API documentation.  Specifically, instead of saying `the user has staff access` we should say `the user has delete access to the other user's profile image`.